doc/_static/xanadu_gallery.css

```diff
@@ -0,0 +1,264 @@
+ 
+/*
```

Comment:
Yep! Will fix this now. This is fixed in most of the other repositories, just not the PennyLane docs (got overlooked). This is a reason why I would like to centralize all our Sphinx CSS, to avoid having to manually update each repository.

Reply:
Fixed